tests/unit/other.tcl

```diff
@@ -574,6 +574,25 @@ start_server {tags {"other external:skip"}} {
     }
 }
 
+start_server {tags {"other external:skip"}} {
+    test "test io-threads are runtime modifiable" {
+        #Test set
+        r config set io-threads 5
```

Comment:
Consider adding a check for cases when we switch between single I/O thread (main thread only) and multiple I/O threads

Reply:
Is just changing the 3 to 1 in this test enough?